### DISCUSSION TITLE
Catalogue

### DISCUSSION THREAD

**Discussion by gndayisenga** [lsc-hubs/rwanda-hub#17] [General]
when searching an Item, why submit is allowing users to search empty or special characters like /,#,*. I think you need to validate the inputs on submit. this is achieved by JavaScript on submit. 
-  Empty values to be checked on submit
- Special characters like %,^,* etc  to be validate, this avoid even sql injection

**Comment by pvgenuchten** [comment]
Hi @gndayisenga, thank you for the suggestion. Sql injection is prevented serverside. Can you provide cases where it is important not to allow searches with special characters? (For example: user accidentally uses special character and then has no search results)
